SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -34,154 +34,74 @@ Worker_ComponentUpdate InterestFactory::CreateInterestUpdate()
 	return CreateInterest().CreateInterestUpdate();
 }
 
+// TODO - timgibson - change API so that we can avoid attaching empty Interest components.
```

Comment:
For TODOs, we normally create associated Jira tickets to track the work.

Reply:
There are a few TODOs added, and if the work could be tackled together, they could all refer to the same ticket.

Reply:
yep. they'll either be fixed or flipped to JIRA before (if) I merge. I wanted to get this change in front of you so we can chat about it tho.